index.js

```diff
@@ -141,7 +141,7 @@ export const startEvaluate = async ({
   })
 
   // Update top measurement stations every 12 hours
-  setInterval(() => updateTopMeasurementParticipants(createPgClient), 1000 * 60 * 60 * 12)
+  setInterval(() => periodicDatabaseRefresh(createPgClient), 1000 * 60 * 60 * 12)
```

Comment:
Do we want this still at 12 hours? I think I had done it before with the leaderboard, so that our 24h cache didn't clear right before it, and then we'd have the same stale data for ~48 hours.

I might be thinking about it wrong though. We could leave it as-is, or switch to 24h, or have two separate intervals, 12h for the leaderboard update, and 24h for the others. What do we think?

Reply:
This depends on the data. Do you think the new functions should run on a different schedule? If there's no obvious issue I would suggest to keep it simple and iterate later.

Reply:
I think it's fine to be on the same schedule, but it does mean that we're running them more often than necessary. Ideally, they should gracefully do nothing though. Now that I think about it, I should add some unit tests to make sure.

Reply:
I did! Just modified the other new tests to do two identical calls and assertions